docs/architecture/cx-3-2/3. Verifiable Credentials/CX-Credentials/Standardized CX-Credential.md

```diff
@@ -0,0 +1,326 @@
+# Summary Credential
+```json
+{
+    "holderIdentifier": "did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
+    "verifiableCredentials":
+    [
+        {
+            "@context":
+            [
+                "https://www.w3.org/2018/credentials/v1",
+                "https://www.w3.org/2018/credentials/examples/v1"
+            ],
+            "id": "urn:uuid:12345678-1234-1234-1234-123456789abc",
+            "type":
+            [
+                "VerifiableCredential",
+                "SummaryCredential"
+            ],
+            "issuer":"did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
+            "issuanceDate": "2023-06-02T12:00:00Z",
+            "expirationDate": "2022-06-16T18:56:59Z",
+            "credentialSubject":
+            {
+                "id": "did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
+                "holderIdentifier": "BPN of holder",
+                "type": "Summary-List",
+                "name": "CX-Credentials",
+                "items":
+                [
+                    "cx-active-member",
+                    "cx-dismantler",
+                    "cx-pcf",
+                    "cx-sustainability",
+                    "cx-quality",
+                    "cx-traceability",
+                    "cx-behavior-twin",
+                    "cx-bpn"
+                ],
+                "contract-templates": "https://public.catena-x.org/contracts/"
+            },
+            "proof":
+            {
+                "type": "Ed25519Signature2018",
+                "created": "2023-06-02T12:00:00Z",
+                "proofPurpose": "assertionMethod",
+                "verificationMethod": "did:web:example.com#key-1",
+                "jws": "eyJhbGciOiJFZERTQSJ9.eyJpYXQiOjE2MjM1NzA3NDEsImV4cCI6MTYyMzU3NDM0MSwianRpIjoiMTIzNDU2NzgtMTIzNC0xMjM0LTEyMzQtMTIzNDU2Nzg5YWJjIiwicHJvb2YiOnsiaWQiOiJkaWQ6d2ViOmV4YW1wbGUuY29tIiwibmFtZSI6IkJlaXNwaWVsLU9yZ2FuaXNhdGlvbiJ9fQ.SignedExampleSignature"
+            }
+        }
+    ]
+}
+
+```
+# BPN Credential 
+```json
+{
+    "id": "UUID",
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/suites/jws-2020/v1",
+        "https://raw.githubusercontent.com/catenax-ng/product-core-schemas/main/businessPartnerData"
+    ],
+    "type": ["VerifiableCredential", "BpnCredentialCX"],
+    "issuer": "<did>",
+    "issuanceDate": "2021-06-16T18:56:59Z",
+    "credentialSubject": {
+        "type":"BpnCredential",
+        "bpn":"bpn"
+    }
+}
+```
+# Membership Credential 
+
+
+
+
+```json
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/suites/jws-2020/v1",
+         "https://raw.githubusercontent.com/catenax-ng/product-core-schemas/main/businessPartnerData"
+    ],
+    "id": "http://example.edu/credentials/58473",
+    "issuer": "<did>",
+    "type": ["VerifiableCredential", "DismantlerCredentialCX"],
```

Comment:
Type should be MembershipCredential

Reply:
This is the Dismantler Credential why should the Type be Membership Credential?